blog/_posts/2050-08-24-aks-past-present-future.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "AKS - How we got here"
+description: "An exploration of the past, present, future for Azure Kubernetes Service (AKS)."
+date: 2050-08-24
+author: Jorge Palma
+categories: general
+---
+
+## Introduction
```

Comment:
Same for this one? Are we going to publish it without being full?

Reply:
These are just placeholders and will be removed when we add the first post. We need something there to render the site and test